ironfist/src/cpp/game/save.cpp

```diff
@@ -456,21 +456,22 @@ void game::LoadGame(char* filnam, int newGame, int a3) {
 			std::map<std::string, mapVariable> mapVariables;
 			for (ironfist_map::map_t::mapVariable_const_iterator it = mp->mapVariable().begin();
 				it != mp->mapVariable().end(); it++) {
-				mapVariable mapVar;
+				mapVariable *mapVar = new mapVariable;

```

Comment:
Do we need to add a destructor somewhere for that object? Or does the map destructor for mapVariables (3 lines above) do that for us?